src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -128,6 +128,11 @@ export class QueryResults {
     return this.tagValues[key];
   }
 
+  hasTags(key: string) {
```

Comment:
```suggestion
  hasTag(key: string) {
```

Reply:
The returned tags are plural.

Reply:
nvm, talked about it, we can maybe change `getTags` to `getTagValues(tag: string)` and this to `hasTag(tag: string)` to make things clearer.